posthog/temporal/data_imports/pipelines/postgres/postgres.py

```diff
@@ -92,6 +93,25 @@ class TableStructureRow:
     numeric_scale: Optional[int]
 
 
+def _get_partition_bucket_size(cursor: psycopg.Cursor, schema: str, table_name: str) -> int | None:
+    query = sql.SQL("""
+        SELECT round(({bytes_per_partition}) / (pg_table_size({schema_table_name_literal}) / count(*)))
```

Comment:
Should we be concerned about `count(*)` being 0 like the bot mentioned? Or have we at this point already established there is some data in the table?

Reply:
Ah yes, I'll add this in now